apps/pie-microsite/src/assets/styles/components/_toggle.scss

```diff
@@ -0,0 +1,116 @@
+$toggle-item-padding-y: 10px;
+$toggle-item-padding-x: f.spacing(e);
+$toggle-item-bg: f.$color-container-strong;
+$toggle-item-bg--hover: darken(f.$color-container-strong, f.$color-hover-01);
+$toggle-item-bg--active: darken(f.$color-container-strong, f.$color-active-01);
+$toggle-item-bg--checked: f.$color-interactive-light;
+$toggle-item-bg--checked-hover: darken(f.$color-interactive-light, f.$color-hover-01);
+$toggle-item-bg--checked-focus: f.$color-interactive-light;
+$toggle-item-bg--checked-active: darken(f.$color-interactive-light, f.$color-active-01);
+$toggle-item-color: f.$color-content-interactive-dark;
+$toggle-item-color--hover: f.$color-content-interactive-dark;
+$toggle-item-color--active: f.$color-content-interactive-dark;
+$toggle-item-color--checked: f.$color-content-interactive-dark;
+$toggle-item-elevation: none;
+$toggle-item-elevation--checked: f.$elevation-02;
+
+.c-toggle {
+    display: inline-flex;
+    align-items: center;
+    flex-wrap: nowrap;
+    padding: $toggle-padding;
+    border-radius: $toggle-radius;
+    background-color: $toggle-bg;
+}
+
+.c-toggle-input {
+    &:focus-visible ~ .c-toggle-label {
+        box-shadow: inset 0 0 0 2px f.$color-focus;
+        outline: 2px solid transparent;
+    }
+
+    position: absolute;
+    display: block;
+    overflow: hidden;
+    // IE fallback
+    clip: rect(1px, 1px, 1px, 1px);
+    // modern browsers
+    clip-path: inset(1px);
+    white-space: nowrap;
+}
+
+.c-toggle-item {
+    z-index: 0; // keep inactive items under active ones
+    height: $toggle-item-height;
+
+    &:hover {
+        z-index: 1; // keep above siblings
+    }
+}
+
+.c-toggle-label {
+    padding: $toggle-item-padding-y $toggle-item-padding-x;
+    @include f.font-size('body-l');
+    position: relative; // context for :before
+
+    align-items: center;
+    justify-content: center;
+
+    width: 100%;
+    height: 100%;
+
+    border-radius: $toggle-radius;
+    border: 0;
+    background-color: $toggle-item-bg;
+    box-shadow: $toggle-item-elevation;
+
+    font-weight: $toggle-item-fw;
+    color: $toggle-item-color;
+    white-space: nowrap;
+    text-align: center;
+
+    cursor: pointer;
+    user-select: none;
+    display: flex;
+
+    .is-active & {
+        background-color: $toggle-item-bg--checked;
+        font-weight: $toggle-item-fw--checked;
+        color: $toggle-item-color--checked;
+        // apply box-shadow separate to prevent overwrite by global focus
+        &:before {
+            content: '';
+            position: absolute;
+            display: block;
+            width: 100%;
+            height: 100%;
+            border-radius: $toggle-radius;
+            box-shadow: $toggle-item-elevation--checked;
+        }
+    }
+}
+
+.c-toggle-item:not(.is-active) {
+    .c-toggle-input:active ~ .c-toggle-label {
+        background-color: $toggle-item-bg--active;
+        color: $toggle-item-color--active;
+    }
+    .c-toggle-label {
+        &:hover {
```

Comment:
:focus?

Reply:
As we mentioned in the other one the designers seem happy with the focus states at the moment, but we will double check